students/erica_edwards/lesson01/assignment/test_integration.py

```diff
@@ -0,0 +1,81 @@
+import sys
+import os
+sys.path.insert(1, f"{os.getcwd()}\\inventory_management")
```

Comment:
This works well, except if the user (or the instructor ;-) ) is using Linux or MacOS, as the folder divider is the slash instead of the backslash. One way to prevent this is to have your code determine which platform type you're running with is to use _os.name_, which will return _posix_ for Unix, Linux or MacOS (which would cover all slash users).